make/setup-cache.mk

```diff
@@ -76,7 +76,7 @@ kubetypes-deps-install:
 		"$(shell go env GOPATH)/src/k8s.io/code-generator"
 
 devdeps-install: kubetypes-deps-install
-	$(GO) install github.com/vektra/mockery/.../
+	$(GO) get     github.com/vektra/mockery/.../
```

Comment:
I think this would install the wrong version of mockery, we're on 1.x from what I recall

Reply:
The version is in `go.mod`, I think it works.  I also tho that they're using a `v2` path for v2.   Could be wrong tho.